scripts/diff-csv.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eu
+
+#
+# This script compares the upstream ClusterServiceVersion with the downstream ClusterServiceVersion.
+# It only prints the diff output, the release manager must verify the changes between upstream and downstream.
+# This is helpful if upstream changed the ClusterServiceVersion in a way which requires downstream changes to the patch script.
+#
+
+DIR="${PWD##*/}"
+
+if [[ "$DIR" = "konflux-jaeger" ]]; then
```

Comment:
This could be perhaps simplified. build the bundle image once, the bundle name differs but that should not matter much